vcx/ci/Jenkinsfile

```diff
@@ -118,82 +117,32 @@ def android() {
         stage('iOS Build') {
             try {
                 checkout scm
-                sh 'chmod +x vcx/ci/scripts/iosBuild.sh'
-                sh './vcx/ci/scripts/iosBuild.sh'
-                /*
-            // 1) Login to a new account on a macOS High Sierra (10.13.4) computer.
-            // ASSUMING THE LOGIN NAME IS: iosbuild1
-
-            // 2) Make sure that the oracle java SDK is installed on the mac
-                sh "brew update"                
-                sh "brew cask install java8"                
-
-            // 3) Checkout the sdk project using https://github.com/evernym/sdk.git or git@github.com:evernym/sdk.git
-                checkout scm
-                sh "brew install git"
-
-            // 4) Copy the sdk/vcx/libvcx/build_scripts/ios/mac/.bash_profile file to your home directory /Users/[username] and replace the
-            // username iosbuild1 with your username.
-
-            // 5) Re-start your terminal/iterm so that the bash settings take effect
-
-            // 6) Install Android Studio or start Android Studio to make sure that the Android sdk
-            // is installed at /Users/[username]/Library/Android/sdk (YES, this is for iOS but we make sure Android Studio is installed)
-                sh "brew install ruby"
-                sh "brew install curl"
-                //ruby -e "$(curl -fsSL https://raw.github.com/Homebrew/homebrew/go/install)"
-                sh "brew doctor"
-                sh "brew install android-sdk"
-
-            // 7) Startup a terminal and cd into sdk/vcx/libvcx/build_scripts/ios/mac
-                sh "cd sdk/vcx/libvcx/build_scripts/ios/mac"
-
-            // 8) Run the script ./mac.01.libindy.setup.sh (make sure the brew install commands are successful)
-                sh "./mac.01.libindy.setup.sh"
-
-            // 9) Restart your terminal for environment variables to take effect and cd into sdk/vcx/libvcx/build_scripts/ios/mac
-            // 10) Run the script 'source ./mac.02.libindy.env.sh'
-                sh "./mac.02.libindy.env.sh"
-
-            // 9) Restart your terminal for environment variables to take effect and cd into sdk/vcx/libvcx/build_scripts/ios/mac
-            // 11) Run the script ./mac.03.libindy.build.sh
-                sh "./mac.03.libindy.build.sh"
-
-            // 12) Run the script ./mac.04.libvcx.setup.sh
-                sh "./mac.04.libvcx.setup.sh"
+                SCRIPTS_PATH="vcx/libvcx/build_scripts/ios/mac"
+                WORK_DIR = "/Users/jenkins"
 
-            // 13) Run the script 'source ./mac.05.libvcx.env.sh'
-                sh "./mac.05.libvcx.env.sh"
+                // Build ios architectures
+                sh "source vcx/ci/scripts/iosBuild.sh"
 
-            // 14) Run the script ./mac.06.libvcx.build.sh (Test failures do not prevent the .a files from being correctly built)
-            // Check for error
-                sh "./mac.06.libvcx.build.sh"
+                // Publish to Kraken
```

Comment:
Can this be moved into only when we are on master branch?

Reply:
No because soon ios tests will be run too and it will need those builds. I have made the publishing happen only on master.